Fixed broken link in README (#9929)
eclipse-theia/theia: packages/keymaps/README.md

You can also use the following strings for special keys: `backspace`, `tab`, `enter`, `return`, `capslock`, `esc`, `escape`, `space`, `pageup`, `pagedown`, `end`, `home`, `left`, `up`, `right`, `down`, `ins`, `del` and `plus`.

> If unsure you can always look at [keys.ts](../core/src/browser/keyboard/keys.ts#515) to see if a string is supported.

Reviewer: The only thing I'm not sure about is that this line number will constantly change.
Should we refer to documentation instead?
- https://eclipse-theia.github.io/theia/docs/next/modules/core.key-2.html

Author: we definitily should!

Author: (done)

> If unsure you can always look at [keys.ts](https://eclipse-theia.github.io/theia/docs/next/modules/core.key-2.html) to see if a string is supported.

Reviewer: Since we no longer point to `keys.ts` directly?

```suggestion
If unsure you can always look at the framework's [supported keys](https://eclipse-theia.github.io/theia/docs/next/modules/core.key-2.html)
```
